tools/pre-commit

```diff
@@ -86,9 +98,17 @@ then
   $JSHINT_EXECUTABLE $JSHINT_ARGS $js_changed_files
   jshint_result=$?
 
-  if [ $gjslint_result -ne 0 -o $jshint_result -ne 0 ] ; then
+  eslint_result=0
+  if [ -r ".eslintrc" ];
+  then
+    $ESLINT_EXECUTABLE $ESLINT_ARGS $js_changed_files
+    eslint_result=$?
+  fi
+
+  if [ $gjslint_result -ne 0 -o $jshint_result -ne 0 -o $eslint_results -ne 0 ] ; then
```

Comment:
The hook doesn't work properly because.... you put a "s" at the end of `$eslint_result` in the comparison ;)